.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
i think you will have to use **/node_modules/** as it doesn't look like they were ignored

Reply:
I thought it worked when the folder looked grayed-out on vsc. Dang. I'll try this again.